accession-commons-mongodb/src/main/java/uk/ac/ebi/ampt2d/commons/accession/persistence/mongodb/repository/BasicMongoDbAccessionedCustomRepositoryImpl.java

```diff
@@ -100,7 +100,7 @@ private void setAuditCreatedDate(Iterable<DOCUMENT> documents) {
     private Optional<String> reportBulkOperationException(BulkWriteError error) {
         if (11000 == error.getCode()) {
             final String message = error.getMessage();
-            Pattern pattern = Pattern.compile("_id_ dup key:.\\{.:.\"(.*)\".\\}");
+            Pattern pattern = Pattern.compile("_id_ dup key:.\\{.*:.\"(.*)\".\\}");
```

Comment:
Is the error message changing in Mongo V6 ?

Reply:
Indeed, it now actually shows the name of the field that's duplicated as opposed to just a space.

Mongo 4 (from EVA-3629):
`message='E11000 duplicate key error collection: eva3096_accession_sharded.submittedVariantOperationEntity index: _id_ dup key: { : "RSMC_GCA_905237065.2_81B5B1F8AF2C566855544E671A7D90875049AE1C" }'`

Mongo 6 (from unit test):
`message='E11000 duplicate key error collection: accession-test.testDocument index: _id_ dup key: { _id: "h1" }'`